# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1296480966 18000
#      Mon Jan 31 08:36:06 2011 -0500
# Node ID 7b12892ae589e25754fa8e57f765705ed91a3250
# Parent  96f33e266b94d28b1d15b7df8b11acd307ebb60a
fix list of files in arpack/module.mk

diff --git a/libcruft/ChangeLog b/libcruft/ChangeLog
--- a/libcruft/ChangeLog
+++ b/libcruft/ChangeLog
@@ -1,8 +1,14 @@
+2011-01-31  John W. Eaton  <jwe@octave.org>
+
+	* arpack/module.mk (EXTRA_DIST): Include arpack/module.mk in the
+	list.  Remove arpack/src/module.mk and arpacck/util/module.mk
+	from the list.
+
 2011-01-31  John W. Eaton  <jwe@octave.org>
 
 	* arpack/LICENSE: Update to current version from
 	http://www.caam.rice.edu/software/ARPACK.
 	* arpack/ARPACK-license-question.email: Delete as it no longer
 	applies to the current license terms.
 	* arpack/RiceBSD.txt: Delete.
 	* arpack/module.mk: Update for license changes.
diff --git a/libcruft/arpack/module.mk b/libcruft/arpack/module.mk
--- a/libcruft/arpack/module.mk
+++ b/libcruft/arpack/module.mk
@@ -1,22 +1,21 @@
 EXTRA_DIST += \
   arpack/LICENSE \
   arpack/README \
+  arpack/docs/README \
   arpack/docs/debug.doc \
   arpack/docs/ex-complex.doc \
   arpack/docs/ex-nonsym.doc \
   arpack/docs/ex-sym.doc \
-  arpack/docs/README \
   arpack/docs/stat.doc \
-  arpack/src/module.mk \
+  arpack/module.mk \
   arpack/src/debug.h \
   arpack/src/stat.h \
-  arpack/src/version.h \
-  arpack/util/module.mk
+  arpack/src/version.h
 
 libcruft_la_SOURCES += \
   arpack/src/cgetv0.f \
   arpack/src/cnaitr.f \
   arpack/src/cnapps.f \
   arpack/src/cnaup2.f \
   arpack/src/cnaupd.f \
   arpack/src/cneigh.f \
